src/main/resources/com/savi/geo/timezones/README.txt

```diff
@@ -0,0 +1,2 @@
+The shapefile files tz_world.shp, tz_world.shx, and tz_world.dbf are taken from
```

Comment:
Just as a question: did you grab new versions of the files? The diff shows just a rename with no changes, and I'm not sure if we started with the most up to date stuff.